packages/gqltest/gqltest.js

```diff
@@ -132,8 +145,8 @@ async function execute({
 //
 // (1) value rooted at `data`: {customer: {name: "Fred"}}
 // (2) root value with no errors: {data: {customer: {name: "Fred"}}}
-// (3) not implemented - root value with field errors: {data: {customer: {name: "Fred" email:null}}, "errors":[...]}
-// (4) not implemented - root value with request errors: {"errors":[...]}
+// (3) root value with field errors: {data: {customer: {name: "Fred" email:null}}, "errors":[...]}
+//     or request errors: {"errors":[...]}
```

Comment:
Responses with request errors are different to responses with field errors, which is why they were separated. Please keep the separation.